libs/domain/site/notification-center/src/notification-center.component.ts

```diff
@@ -31,13 +32,9 @@ export class SiteNotificationCenterComponent extends ContentMixin<SiteNotificati
       if (!(this.centerRef.value && 'open' in this.centerRef.value)) {
         await customElements.whenDefined('oryx-notification-center');
       }
-      if (
-        !notification.autoCloseTime &&
-        this.componentOptions?.defaultAutoCloseTime
-      ) {
-        notification.autoCloseTime =
-          this.componentOptions.defaultAutoCloseTime * 1000;
-      }
+      notification.autoCloseTime ??=
+        (this.componentOptions.defaultAutoCloseTime ??
```

Comment:
Why do we need to use `options.defaultAutoCloseTime` as fallback value? If user doesn't provide defaultAutoCloseTime in custom config - it will use `defaultAutoCloseTime` from default options by default.